e2e-tests/manifests/vapi-deployment.yaml

```diff
@@ -17,13 +17,24 @@ spec:
         app: vapi
     spec:
       containers:
-      # command is custom because:
```

Comment:
@sfowl I've enabled the VAPI frontend, so we can test `spiderAjax` with it. I assume it wasn’t required for earlier versions of the E2E tests, but please take a look in case I’ve missed something